Makefile

```diff
@@ -55,3 +55,12 @@ vet: ## Run go vet against code.
 .PHONY: test
 test: fmt vet ## Run tests.
 	go test ./... -coverprofile cover.out
+
+
```

Comment:
This is not really a problem with the makefile. But I did not know where to put the comment. We can update the README.md to say that you can run `make test` to run the tests. Currently it has some other commands.